pkg/providers/snow/reconciler/reconciler.go

```diff
@@ -48,5 +49,25 @@ func (r *Reconciler) Reconcile(ctx context.Context, log logr.Logger, c *anywhere
 		}
 	}
 
+	kubeClient := clientutil.NewKubeClient(r.client)
+	if result, err := r.reconcileControlPlaneSpec(ctx, c, specWithBundles, kubeClient, log); err != nil {
+		return result, err
+	}
+
+	return controller.Result{}, nil
+}
+
+func (r *Reconciler) reconcileControlPlaneSpec(ctx context.Context, cluster *anywherev1.Cluster, specWithBundles *cluster.Spec, kubeClient *clientutil.KubeClient, log logr.Logger) (controller.Result, error) {
+	log.Info("Applying control plane spec", "name", cluster.Name)
+	objs, err := snow.ControlPlaneObjects(ctx, specWithBundles, kubeClient)
+	if err != nil {
+		return controller.Result{}, err
+	}
+	for _, obj := range objs {
+		key := client.ObjectKey{Name: obj.GetName(), Namespace: obj.GetNamespace()}
+		if err := r.client.Get(ctx, key, obj); err != nil {
```

Comment:
I might be missing something, but what is this supposed to do? Shouldn't we be creating/updating the objects?